configure.ac

```diff
@@ -89,6 +89,7 @@ AS_IF([test "x${enable_flags_setting}" = "xyes" && test "x${enable_debug}" = "xn
 
 	AC_LANG([C++])
 	AX_APPEND_COMPILE_FLAGS([${cflags_test}], [CXXFLAGS])
+	AX_CXX_COMPILE_STDCXX([11], [ext], [mandatory])
```

Comment:
why are you enabling GNUisms?

Reply:
`struct` members initialization is only available in C++20,
but sadly, several environment, such as bioconda are stuck with an old GCC 4.9 that is barely C++11 (it even misses the regex from libstdc++)

enabling the GNU-ism is a work around that works on both GCC and LLVM.

Reply:
what struct member initializer? what about aggregate initialization?

Reply:
What `struct` initializers? Do you mean `designated initializers`? If so, either use aggregate initialization or define a constructor. Using GNUisms for that is not a win.